src/pages/user/Messages.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import {
+  Box,
+  Typography,
+  Paper,
+  Divider,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemAvatar,
+  Avatar,
+} from '@mui/material';
+import { blue, grey } from '@mui/material/colors';
+
+const MessageThread = ({ messageId }: { messageId: number }) => {
+  // Simulated message thread data
+  const messageThread = [
```

Comment:
We need to take data from Database but not sample data on the page, this part that you need to continue working on Sprint3